rear_camera_py/rear_camera/line_angle_calculator.py

```diff
@@ -226,116 +167,143 @@ def get_transformed_image(self, img: np.ndarray, borderValue=(255, 255, 255)) ->
         return cv2.warpPerspective(
             img, self.__trans_mat, (img.shape[1], img.shape[0]), borderValue=borderValue)
 
-    @staticmethod
-    def get_masked_image(img: np.ndarray) -> np.ndarray:
-        """画像上の円(交点)を検出し、大きめの円で上書きする(マスク処理を行った画像を作成).
-
-        四隅の交点周りのラインが交わっているため、補正する際に2本の線が1本として認識される.
-        円(交点)を大きな円で上書きすることで交わっている部分を隠す.
+    def detect_line_segment(
+        self,
+        img,
+        length_threshold: int = 50,
```

Comment:
線分を検出する際の長さを`mm`で指定するのはどうでしょうか？
後から値を調整したくなった際に、より直感的に指定できると思います。

```suggestion
        length_threshold_mm: int = 50,
```